test/ode_tests_common.jl

```diff
@@ -12,16 +12,16 @@ const explicit_methods = (
     (LSRK54CarpenterKennedy(), 4),
     (LSRK144NiegemannDiehlBusch(), 4),
     (LSRKEulerMethod(), 1),
-    (SSPRK22Heuns(), 2),
-    (SSPRK22Ralstons(), 2),
-    (SSPRK33ShuOsher(), 3),
-    (SSPRK34SpiteriRuuth(), 3),
+    # (SSPRK22Heuns(), 2),
+    # (SSPRK22Ralstons(), 2),
+    # (SSPRK33ShuOsher(), 3),
+    # (SSPRK34SpiteriRuuth(), 3),
```

Comment:
Is this meant to be commented?

Reply:
Yes because those methods are gone now and the old tests don't work with ClimaODEFunction